### DISCUSSION TITLE
Blocklists

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#2532] [Q&A]
Scouring around, there is a lot of outdated information about Transmission and blocklists around, and little to do with the creation of your own blocklist, how it should be formatted etc. Does anyone have some good working knowledge of adding your own blocklists outside of the URL-based ones? On MacOS?

**Comment by jensche** [comment]
I'm using the following Blocklist: 
http://list.iblocklist.com/?list=bt_level1&fileformat=p2p&archiveformat=gz

But is there a better one?

**Reply by GaryElshaw** [reply]
My go-to: https://github.com/Naunter/BT_BlockLists
